Sources/Web3Auth/Web3Auth.swift

```diff
@@ -153,11 +153,6 @@ public class Web3Auth: NSObject {
      - parameter callback: Callback called with the result of the WebAuth flow.
      */
     public func login(_ loginParams: W3ALoginParams) async throws -> Web3AuthState {
-        // Think we should avoid this and throw error instead
```

Comment:
Perhaps this instead? Similar for the other one.

```
if initParams.redirectUrl == nil {
   // throw error here
}
```

Reply:
Since bundleid was originally being used to ensure redirecturl was not nil.

Reply:
added the checks now